templates/dell_force10_show_ip_interface_brief.textfsm

```diff
@@ -0,0 +1,9 @@
+Value INTERFACE (\S+ \S+)
```

Comment:
Do you mind truing up the values to be more aligned with https://github.com/networktocode/ntc-templates/blob/master/templates/cisco_ios_show_ip_interface_brief.textfsm so it can be somewhat vendor neutral for the overlapping fields. I would change `OK` to `STATUS` most likely as well.

Also, if you look at the readme, we have some details on what we're expecting from contributions. Main things are the raw output and the parsed version of it as well.

Reply:
I have changed this, `STATUS` is already present as a variable so I decided to keep the `OK` variable as was.

Reply:
We just need to add tests now. Here is some details on how to add tests: https://github.com/networktocode/ntc-templates#tests

You can join our Slack community as well if you want to talk live.

Reply:
It has been fixed.